demo/demo.html

```diff
@@ -7,7 +7,7 @@
 	<base href=".."></base>
 
 	<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.0.8/angular.min.js"></script>
-	<script src="http://code.jquery.com/jquery-1.10.1.min.js"></script>
+	<script src="https://code.jquery.com/jquery-1.10.1.min.js"></script>
```

Comment:
Nice.  The sweeping standardization of [network-path reference](https://tools.ietf.org/html/rfc3986#section-4.2) is new to me (I've been doing this for a long time; I actually only have memories of that failing to work!)  Still, given the previously loaded script is SSL and the security risks of a [MitM](https://en.wikipedia.org/wiki/Man-in-the-middle_attack) or [other](https://en.wikipedia.org/wiki/Man-on-the-side_attack) attacks (it's a script, after all), this is ill advised.

Even Paul Irish, atop the very page you're citing, says that "**this technique is now an anti-pattern**. If the asset you need is available on SSL, then **always** use the `https://` asset" (emphasis is his).